server/app.js

```diff
@@ -1,13 +1,29 @@
 import express from 'express';
-import mongoose from 'mongoose';
 import bodyParser from 'body-parser';
+import Sequelize from 'sequelize';
 import routes from './routes';
 
-mongoose.connect('mongodb://localhost:27017/redditclone', () =>{
-    console.log("connected to mongodb");
+const app = express();
+
+var sequelize = new Sequelize('athena_development', 'root', 'led107a100', {
+  host: 'localhost',
+  dialect: 'mysql',
+  pool: {
+    max: 5,
+    min: 0,
+    idle: 10000
+  }
 });
 
-const app = express();
+sequelize
+  .authenticate()
+  .then(function(err) {
+    console.log('Connection has been established successfully.');
```

Comment:
does Node's default console provide a debug logging level?

If not, we can add the Winston logger later